web-dev/src/components/HomePage/home-page.css

```diff
@@ -2,6 +2,7 @@
     height: 150vh;
```

Comment:
This isn't really used. I'm using it to pad the page so that it's tall in order to be able to scroll so that the nav bar on scroll event can trigger.